client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -143,21 +143,105 @@
     "description": "Feature Flag related to Codegen",
     "features": {
       "useAppsyncModelgenPlugin": {
```

Comment:
1. Why is the flag called `...Appsync` if we're referencing DataStore? Not really sure reading the docs what this enables
2. Here and throughout: "Allows to use..." is a bit cumbersome. Shorten to just "Use..."

Reply:
The naming of that flag was CLI team's choice and has already been released previously. I believe it is inspired from https://github.com/aws-amplify/amplify-cli/tree/master/packages/amplify-codegen-appsync-model-plugin.